build.rs

```diff
@@ -77,7 +77,10 @@ fn build_ossl(out_file: &Path) {
             "no-ec2m",
             "no-sm2",
             "no-sm4",
-            "-DREDHAT_FIPS_VERSION=\\\"0.0.1-test\\\"",
+            "no-des",
+            "no-dsa",
+            "no-atexit",
+            "-DDEVRANDOM=\\\"/dev/urandom\\\" -DOPENSSL_PEDANTIC_ZEROIZATION -DFIPS_VENDOR=\\\"Kryoptic\\\" -DKRYOPTIC_FIPS_VERSION=\\\"1.0.0-test\\\"",
```

Comment:
should we take the version from the package version here to avoid the need to update version in different places?

Reply:
For upstream that is ok, yes, downstream we'll probbaly do something different anyway.